profile_collection/startup/99-subtracted-tiff-exporter.py

```diff
@@ -51,5 +19,9 @@ def event(self, doc):
         doc['data'][self.field] = subtracted_img
         super().event(doc)
 
-template = "/home/xf28id1/xpdUser/tiff_base/LaB6_EPICS/{start.sa_name}_{start.scan_id}_step{event.seq_num}.tif"
+
+# TODO this looks too hard-coded!
+template = ("/home/xf28id1/xpdUser/tiff_base/LaB6_EPICS/"
+            "{start.sa_name}_{start.scan_id}_step{event.seq_num}.tif")
```

Comment:
Should we move to a `{start[sa_name]}` format since we will get dicts going forward?